CONTRIBUTING.md

```diff
@@ -146,3 +146,21 @@ and triggers release and publishing based on the following rules:
 | Commit with type `feat`            | Minor release |
 | Commit with type `fix`             | Patch release |
 | Commit with type `perf`            | Patch release |
+
+## Link widgets with local version of components/sdk-component-adapter
```

Comment:
@lalli-flores this requires your review to make sure the explanation is clear and in proper English.